skill_tagging/static/tagging.js

```diff
@@ -0,0 +1,68 @@
+const tagSkillContainer = document.getElementById("tag-verification-tags-container-id");
+var tagSkillSelectedTags = [];
```

Comment:
Nit: For the sake of brevity, perhaps the variable should be named `tagSkillVerifiedTags`?